labgrid/driver/power/poe_mib.py

```diff
@@ -0,0 +1,27 @@
+""" tested with Cisco CBS350, should be compatible with switches implementing the PoE administration MiB"""
+
+from ..exception import ExecutionError
+from ...util.snmp import SimpleSNMP
+
+OID = "1.3.6.1.2.1.105.1.1.1.3.1"
+
+def power_set(host, port, index, value):
+    _snmp = SimpleSNMP(host, 'private')
```

Comment:
Shouldn't we pass the port here?

Reply:
Since the devices are network switches where SNMP may already be enabled, I don't see the value in a configurable port.

Reply:
You can configure the "snmp-agent-port" on the Cisco switches, so it should also be possible to configure it on labgrid's side. The other power backends using SNMP allow the same.